src/processes.c

```diff
@@ -946,7 +950,8 @@ static void ps_submit_fork_rate (derive_t value)
 
 /* ------- additional functions for KERNEL_LINUX/HAVE_THREAD_INFO ------- */
 #if KERNEL_LINUX
-static procstat_t *ps_read_tasks_status (long pid, procstat_t *ps)
+static procstat_gauges_t *ps_read_tasks_status (long pid,
+                                                procstat_gauges_t *gauges)
```

Comment:
You might as well s/`gauges`/`g`/. I know short names aren't great, but using the full `gauges` doesn't buy us anything either. Also in other functions.

Reply:
Done.